### DISCUSSION TITLE
Border radius of code cell

### DISCUSSION THREAD

**Discussion by Bomma-Pranay** [jupyterlab/jupyterlab#14830] [Q&A]
Hi, I was wondering if there is a way to customize the border radius of the code cell in Jupyter?

**Comment by krassowski** [comment]
Certainly! There is a number of themes which do that, e.g. https://github.com/simicd/jupyterlab-tailwind-theme or https://github.com/martinRenou/jupyterlab-night. You could build your own theme using https://github.com/jupyterlab/extension-template or try using https://github.com/jupyterlab-contrib/jupyterlab-theme-editor (but I think you would need to go to the CSS files to set the border radius in the end).

**Reply by Bomma-Pranay** [reply]
Thanks for the reply Mike. Is there any option for me to use the default theme but only change the code cell border radius using some CSS?

**Reply by krassowski** [reply]
Your custom themes can inherit from a default theme. There is no officially supported way to customise CSS other than by themes, but extensions can ship any CSS they like (so one could add an extension which overrides border radius and exposes a setting). Or you could just add raw CSS via settings of some extensions which allow it like https://github.com/deathbeds/jupyterlab-fonts (does not support lab 4.0 yet).

**Reply by krassowski** [reply]
I mean there probably will be a way in JupyterLab 4.1, see https://github.com/jupyterlab/jupyterlab/pull/14743 but for now themes of jupyterlab-fonts are the only ways.
